.github/workflows/cicd.yml

```
@@ -0,0 +1,175 @@
+        - name: Checkout
+          uses: actions/checkout@v4
+          with:
+            fetch-depth: 2
+
+        # 2. 변경된 service 감지
+        - name: Detect Changed Services
+          id: detect_changes
+          run: |
+            CHANGED_FILES=$(git diff --name-only HEAD^ HEAD)
+            echo "Changed files: $CHANGED_FILES"
+            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)